scripts/function_check_if_existing_comment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Define the check_if_existing_comment function
+function check_if_existing_comment {
+    echo "Checking if existing comment..."
+
+    local vale_json="$1"
+    #local pull_comment_json="$2"
+
+    # Outer loop to iterate through Vale alert JSON
+    echo "$vale_json" | jq -c '.[]' | while read object; do
```

Comment:
If we're passing valid json, don't need the '.[]' I think

```suggestion
    echo "$vale_json" | jq | while read object; do
```